playwright.config.ts

```diff
@@ -4,9 +4,10 @@ const config: PlaywrightTestConfig = {
   testDir: './e2e-tests',
   maxFailures: 1,
   workers: 1, // NOTE: must be 1 for now, otherwise tests will fail.
+  retries: process.env.CI ? 2 : 0, // Retry on CI only.
```

Comment:
Random tests fail due to timeout requiring restarting the CI pipeline to re-run the tests. Adding this configuration option instructs Playwright to retry when a test fail.

Reply:
@cnrpman not sure what OT stands for 🤔. I'll revert this change although I think there is no harm in leaving it in.